krizten/Memoire-WebAPI
# #164897946 User Signup Endpoint for Memoire API

### What does this PR do?
* Built out endpoint to signup user
#### Description of Task to be completed?
Get the following endpoint working
```POST /api/v1/auth/signup ```
#### How should this be manually tested?
1. Clone the repository, install the dependencies and run npm run start:dev.
2. Using a REST client such as Postman, test the endpoint to ensure it gives result.
#### What are the relevant pivotal tracker stories?
#164897946
